### ISSUE TITLE
cannot deselect chosen group categories

### ISSUE THREAD

**Issue by NouranHossam** [GeoNode/geonode#3991]
Hello 
I am using GeoNode 2.8 and I am trying to create new groups and attach a category for each group.
The problem i am facing is that when I cannot deselect a clicked category, if i selected a category by wrong and i want to deselect i cannot.
![image](https://user-images.githubusercontent.com/25221943/46960245-630ce600-d09e-11e8-97f0-5731f1242545.png)

**Comment by t-book** [comment]
I don't think this is a bug. You can select/deselect an option in an multiple html select by holding down the command key. On Mac it´s the one with the apple sign on windows it might be the one .. well yes the key with the windows logo (or ctrl).

**Comment by t-book** [comment]
@NouranHossam Can you check if this works for you otherwise @afabiani  let´s close this?

**Comment by t-book** [comment]
@francbartoli I think this can be closed ? (answer is given and questioner seems non active!)

**Comment by francbartoli** [comment]
@t-book I have just doubts cause @afabiani have labeled it

**Comment by t-book** [comment]
Ok thanks Francesco. Let´s wait.

**Comment by t-book** [comment]
@afabiani I think this issue should be solved with my given answer, further the questioner seams to be inactive? Let´s close this?
